bridgeifns.py

```diff
@@ -0,0 +1,348 @@
+    if status == False:
+        return status, msg
+
+    return True, messages[1100]
+
+def read(
+    bridgename: str,
+    namespace: str,
+    config_file=None
+) -> Tuple[bool, dict, str]:
+    """
+    description:
+        reads a namespace.bridgename interface from namespace.
+
+    parameters:
+        bridgename:
+            description: The name of the bridge associated with the interface.
+            type: string
+            required: true
+        namespace:
+            description: The VRF network namespace identifier, such as 'VRF123'.
+            type: string
+            required: true
+
+    return:
+        description: |
+            A tuple with a boolean flag indicating if the interface was successfully read,
+            and the output or error message.
+        type: tuple
+    """
+
+    # Define message
+    messages = {
+        1200: f'1200: Successfully removed interface {namespace}.{bridgename} inside namespace {namespace}',
+        1201: f'1201: Interface {namespace}.{bridgename} does not exist',
+
+        3121: f'3021: Failed to connect to the enabled PodNet from the config file {config_file} for payload interface_check:  ',
+        3122: f'3022: Failed to connect to the enabled PodNet from the config file {config_file} for payload interface_del:  ',
+        3123: f'Failed to run interface_del payload on the enabled PodNet. Payload exited with status ',
+
+        3151: f'3021: Failed to connect to the disabled PodNet from the config file {config_file} for payload interface_check:  ',
+        3152: f'3022: Failed to connect to the disabled PodNet from the config file {config_file} for payload interface_del:  ',
+        3153: f'Failed to run interface_del payload on the disabled PodNet. Payload exited with status ',
+    }
+
+    # Default config_file if it is None
+    if config_file is None:
+        config_file = '/opt/robot/config.json'
+
+    status, config_data, msg = load_pod_config(config_file)
+    if not status:
+      if config_data['raw'] is None:
+          return False, None, msg
+      else:
+          return False, msg + "\nJSON dump of raw configuration:\n" + json.dumps(config_data['raw'],
+              indent=2,
+              sort_keys=True)
+    enabled = config_data['processed']['enabled']
+    disabled = config_data['processed']['disabled']
+
+    # Define payload
+
+    def run_podnet(podnet_node, prefix, successful_payloads, data_dict):
+        retval = True
+        data_dict[podnet_node] = {}
+
+        rcc = SSHCommsWrapper(comms_ssh, podnet_node, 'robot')
+        fmt = PodnetErrorFormatter(
+            config_file,
+            podnet_node,
+            podnet_node == enabled,
+            {'payload_message': 'STDOUT', 'payload_error': 'STDERR'},
+            successful_payloads
+        )
+
+        payloads = {
+                'interface_show': f'ip netns exec {namespace} ip link show | grep {namespace}.{bridgename}'
```

Comment:
```suggestion
                'interface_show': f'ip netns exec {namespace} ip link show | grep --word "{namespace}.{bridgename}"'
```
--word to grep the exact word